ui/src/app/components/gke-app-configuration-panels/create-gke-app.tsx

```diff
@@ -181,6 +222,31 @@ export const CreateGkeApp = ({
       }
     : {};
 
+  const canConfigureMachineType =
+    enableGKEAppMachineTypeChoice &&
+    !isAppActive(app) &&
+    otherApps(userApps, appType).length === 0;
+
+  const machineTypeDisabledText = cond(
+    [
+      otherApps(userApps, appType).length > 0,
+      'Cannot configure the compute profile when environments already exist in the workspace.  ' +
```

Comment:
the text in JIRA is “You cannot modify machine type with another application running. To customize this application machine, you have to delete your other application.”

What do you think of this version?

Reply:
Setting aside the bigger discussion for another day... I see what you mean about environments vs. applications.  I'll try some different text which uses "application"